### PR TITLE
add new functions from the python math module

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/column_maker/column_maker.xml [hunk 1]
```diff
@@ -283,6 +288,7 @@ ${expr.cond};${expr.add_column.pos}${expr.add_column.mode};
             <assert_stderr>
                 <has_text text="division by zero" />
             </assert_stderr>
+
```
Comment (nsoranzo) [inline tools/column_maker/column_maker.xml:291]:
```suggestion
```

File: tools/column_maker/column_maker.py [hunk 2]
```diff
@@ -13,12 +13,62 @@
...
+    atanh,
+    cbrt,
     ceil,
+    comb,
+    copysign,
+    cos,
+    cosh,
+    degrees,
+    dist,
+    efabs,
```
Comment (nsoranzo) [inline tools/column_maker/column_maker.py:31]:
```suggestion
```

File: tools/column_maker/column_maker.py [hunk 3]
```diff
@@ -13,12 +13,62 @@
...
+    erf,
+    erfc,
     exp,
+    exp2,
+    expm1,
```
Comment (nsoranzo) [inline tools/column_maker/column_maker.py:36]:
```suggestion
    expm1,
    fabs,
```

File: tools/column_maker/column_maker.xml [hunk 4]
```diff
@@ -34,8 +34,8 @@
         </xml>
     </macros>
     <requirements>
-        <requirement type="package" version="3.8">python</requirement>
-        <requirement type="package" version="1.23.1">numpy</requirement>
+        <requirement type="package" version="3.11">python</requirement>
```
Comment (wm75) [inline tools/column_maker/column_maker.xml:37]: why not 3.12 already?

File: tools/column_maker/column_maker.py (hunk 2 above)
Comment (bgruening) [reply tools/column_maker/column_maker.py:31]: There seems to be a few more gone with the new python version. Will update.

**Comment by wm75** [comment]
maybe better to just not import math.e?
Single-letter variables are just too common and it also creates sort of a combinatorial effect on the whitelist regex if we start allowing additional single characters.

math.e can also easily be obtained as math.exp(1) if you really need the value.

**Comment by bgruening** [comment]
@wm75 done!

**Review by wm75 — APPROVED** [review]
Very nice enhancement, thanks!

**Pull request merged**